src/shared/selinux-util.c

```diff
@@ -33,10 +33,16 @@
 DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(context_t, context_free, NULL);
 #define _cleanup_context_free_ _cleanup_(context_freep)
 
+typedef enum Initialized {
+        UNINITIALIZED,
```

Comment:
this field should be named OFF, I think because it is more than just "uninitialized". It also means we should leave it off permanently until we are told otherwise.

Reply:
or to say this differently: if i read this i would assume this is the state that we in until we lazily initialize things. but that's not the case.